mev-relay-rs/src/relay.rs

```diff
@@ -607,3 +615,18 @@ impl BlindedBlockRelayer for Relay {
         Ok(())
     }
 }
+
+fn verify_blinded_block_signature(
+    context: &Context,
+    auction_request: &AuctionRequest,
+    signed_block: &mut SignedBlindedBeaconBlock,
```

Comment:
nitpick but generally put `context` as the last fn arg
```suggestion
    auction_request: &AuctionRequest,
    signed_block: &mut SignedBlindedBeaconBlock,
    context: &Context,
```